LocalFeeder/sender_cosim.py

```diff
@@ -334,6 +337,14 @@ def go_cosim(
     sub_invcontrol.set_default("[]")
     sub_invcontrol.option["CONNECTION_OPTIONAL"] = 1
 
+    pv_set_key = (
+        "unused/pv_set" if "pv_set" not in input_mapping else input_mapping["pv_set"]
+    )
+
+    sub_pv_set = vfed.register_subscription(pv_set_key, "")
+    sub_pv_set.set_default("[]")
+    sub_pv_set.option["CONNECTION_OPTIONAL"] = 1
```

Comment:
instead of using magic numbers use enums or variables. Allows folks to know what options are available

Reply:
Switched to True. It seemed to work.